components/chainhook-types-rs/src/events.rs

```diff
@@ -121,6 +121,11 @@ pub struct SmartContractEventData {
     pub hex_value: String,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub struct TenureChangeEventData {
+    pub consensus_hash: String,
```

Comment:
We should include the fields from the `tenure_change_payload` example here:
https://api.hiro.so/extended/v1/tx/0x748962f013310e3276eea20cce032af876fe8dc2031367c4e46f5bfa2cd354c0

That should be enough